env-check.sh

```diff
@@ -37,13 +39,24 @@ func_check_file(){
     check_res=1
 }
 
+func_check_exec_binary() {
+    if ! type "$1" &> /dev/null; then
```

Comment:
If you redirect only stderr and not stdout then it will show which binary comes first in the PATH which is useful.

You can avoid the negation with `type ... || {`